functional_testing/Openvcloud/ovc_master_hosted/OVC/a_basic/users_tests.py

```diff
@@ -83,3 +183,28 @@ def test005_create_users_with_same_specs(self):
         #. Create User3 with same Email as User1 , should fail .
 
         """
+
+        self.lg('- Create user1 with random name user1. ')
+        user1_name = self.cloudbroker_user_create()
+        user1_emailaddress = "%s@example.com"%user1_name
+
+        self.lg("- Create User2 with same name as user1, should fail")
+        user2_emailaddress = "%s@example.com"%(str(uuid.uuid4()).replace('-', '')[0:10])
+        try:
+            self.api.cloudbroker.user.create(username=user1_name, emailaddress=user2_emailaddress,
+                                                password=user1_name,groups=[])
+
+        except HTTPError as e:
+            self.lg('- expected error raised %s' % e.status_code)
+            self.assertEqual(e.status_code, 409)
+
+        self.lg("Create User3 with same Email as User1 , should fail . ")
+
+        username3 = str(uuid.uuid4()).replace('-', '')[0:10]
```

Comment:
try to make all variables names in the same format like 
user_name1 user_name3 (not username3)